openlibrary/plugins/openlibrary/home.py

```diff
@@ -47,105 +46,31 @@ def GET(self):
         except:
             loans = None
 
-        return render_template(
+        page = render_template(
             "home/index", stats=stats,
             blog_posts=blog_posts,
             lending_list=lending_list,
             returncart_list=returncart_list,
-            user=user, loans=loans,
-            popular_books=[],
-            waitlisted_books=[]
+            user=user, loans=loans
         )
+        page.homepage = True
+        return page
+
 
+@public
```

Comment:
not an ideal location for this function, but hard to find a good place to put `@public` decorated methods -- that is those to which our templates have access